### ISSUE TITLE
Request: Add nymea as gateway software

### ISSUE THREAD

**Issue by mzanetti** [blakadder/zigbee#567]
nymea (https://nymea.io/users) is a FOSS smart home/IoT platform with good ZigBee support. Would be nice to see it on the list of gateways.
Currently supported ZigBee adapters:
* All phoscon models (ConBee I & II, RaspBee I & II)
* NXP JN5168 & JN5169 (Which seem not to be included in the database yet)
* All serial port based TI z-Stack models (CC253x - zStack 1.3 & CC2652[P|R|RB] - zStack 3x0). Specifically tested with ebyte 2530, Sonoff Zigbee 3.0 USB and the slae.sh dongle (https://slae.sh/projects/cc2652/) which is currently missing too.
I'd submit the missing adapters once nymea can be selected in the google form.

As supported devices, it's well tested with all Hue,Tradfri,Develco models, supports most Lumi/aqara/xiami models and many more, but also implements generic device support as a catch-all to support ZigBee spec compliant devices, so anything that follows the standard Cluster Library should work. I could go through the code and submit all explicitly supported devices as well as listing all known-to-work devices reported by the nymea community.

**Comment by mymike00** [comment]
Yeah, having Nymea listed in the gateways would be very cool! Looking forward an update on this topic 😀
